src/model/XRef.tsx

```diff
@@ -100,13 +100,12 @@ export const XRefList: React.FC<XRefListProps> = ({
         return null;
       }
       return (
-        <ul key={v1()}>
+        <ul className="no-bullet" key={v1()}>
           <XRefItem xRefEntry={xref} accession={accession} />
         </ul>
       );
     });
-  const infoData = [{ title: database, content }];
-  return <InfoList infoData={infoData} />;
+  return <Fragment>{nodes}</Fragment>;
 };
 
 export const XRefCategoryTable: React.FC<XRefCategoryTableProps> = ({
```

Comment:
This is returning an InfoList, not a Table so XrefCategoryInfoList would be more fitting.

Reply:
It's returning more than an InfoList (also an hr and a title). Now refactored so that is returning only an InfoList and updated component name to `XRefCategoryInfoList`.